code/modules/research/designs/engineering.dm

```diff
@@ -162,3 +162,13 @@
 	materials = list(MAT_IRON = 500, MAT_GLASS = 100)
 	build_path = /obj/item/device/t_scanner/advanced
 	category = "Engineering"
+
+/datum/design/experimental_welder
```

Comment:
It's really slow and you risk lighting oil spills and plasma leaks on fire. On tick you have a chance to gain 1 fuel (10%) and a chance to lose 1 fuel (5%) inherited from the parent.

Fuel cans are better in practice, guaranteed refilling, no risk, way faster.